Demo/Tests/SignalProducer+MoyaSpec.swift

```diff
@@ -258,14 +246,12 @@ class SignalProducerMoyaSpec: QuickSpec {
                 let signal = signalSendingData(data as Data)
                 
                 var receivedError: Moya.Error?
-                signal.mapString().start { (event) -> Void in
-                    switch event {
-                    case .next:
+                signal.mapString().startWithResult { result -> Void in
+                    switch result {
+                    case .success:
                         fail("next called for invalid data")
-                    case .failed(let error):
+                    case .failure(let error):
                         receivedError = error
```

Comment:
Do these changes make sense to you, @justinmakaila? I don't want to do something non-idiomatic to ReactiveSwift.